src/contractIntegration/IPFSManagementInteraction/IPFSManagementInteraction.ts

```
@@ -1,3 +1,5 @@
+/*eslint no-unused-vars: "error"*/
+//eslint-disable-next-line
```

@HectorGerardoZV Specify what type of error was ignored

---

I added //eslint-disable-next-line, since it is used @injectable() and does not recognize it, so it takes it as not used {injectable} variable.